python/ql/src/semmle/python/dataflow/new/internal/DataFlowPrivate.qll

```diff
@@ -1237,14 +1234,54 @@ predicate subscriptReadStep(CfgNode nodeFrom, Content c, CfgNode nodeTo) {
  *
  *  `c`: [ListElementContent]
  */
-module UnpackingAssignment {
+module IterableUnpacking {
```

Comment:
I wonder if it would make more sense to move this out of `DataFlowPrivate.qll` and into, say, `Flow.qll`. It seems like a thing that might be useful more generally (and `DataFlowPrivate` is starting to bulge at the seams).

Reply:
This module and `ArgumentPassing` are both quite heavy. We should consider pulling them out, perhaps into files with the module names? Should we do that in a separate PR?

Reply:
Yeah, this is better left as a separate PR.